test/schema/export_test.js

```diff
@@ -21,14 +24,27 @@ describe('#exportToSchemasUniqueCases()', () => {
     let schemas = exportToSchemas([ns]);
     expect(schemas).to.have.length(16);
     expect(schemas).to.eql(expectedSchemas);
+    expect(validateSchema(schemas));
   });
 });
 
 function exportNamespaces(...namespace) {
   let schemas = exportToSchemas(namespace);
+  expect(validateSchema(schemas));
   return schemas;
 }
 
 function importFixture(name, ext='.json') {
   return JSON.parse(fs.readFileSync(`${__dirname}/fixtures/${name}${ext}`, 'utf8'));
 }
+
+function validateSchema(schema) {
+    const v = new validator();
+    const extendedSchema = importSchema();
+    return v.validate(schema, extendedSchema);
```

Comment:
I don't think you're using this function correctly.  Based on the [jsonschema doc](https://github.com/tdegrunt/jsonschema/blob/master/README.md), the first arg is the version of json-schema (`4` in our case) and the second is the thing to validate (`schema` in our case).

This is their example:
```javascript
var Validator = require('jsonschema').Validator;
  var v = new Validator();
  var instance = 4;
  var schema = {"type": "number"};
  console.log(v.validate(instance, schema));
```

Based on the doc, I think that supporting the extended schema happens through their 'customProperties' feature -- which needs to be hand-coded.  As best I can tell, you can't just pass in the extended-schema.  You  have to write customProperty validators for each custom property in the schema.  That's pretty annoying. :-/